modules/hostap/src/supp_main.c

```diff
@@ -42,10 +42,6 @@ LOG_MODULE_REGISTER(wpa_supplicant, LOG_LEVEL_DBG);
 K_SEM_DEFINE(z_wpas_ready_sem, 0, 1);
 #include <l2_packet/l2_packet.h>
 
-/* Should match with the driver name */
-#define DEFAULT_IFACE_NAME "wlan0"
```

Comment:
Please put this changes in a separate commit, a commit shouldn't span multiple modules, makes is tough to cherry pick or revert.

Reply:
Added a separate commit